### DISCUSSION TITLE
12/24 hour timestamp format

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#292] [Q&A]
Hero 8 Black, Using BURN to add a timestamp. According to the description, using aa or AA to add am/pm switches off 24 hour time. When I use either I get the am/pm added, but the time is still in 24hr format. Is this a bug? Anyone else have this problem? 

    HH - Hour
    MM - Minute
    SS - Second
    aa - am/pm (also switches off 24 hour time)
    AA - AM/PM (also switches off 24 hour time)

![image](https://user-images.githubusercontent.com/123906976/229963507-74eb8a79-c5f7-4e45-815c-1cec0545218b.png)

**Comment by dnewman-gpsw** [comment]
Yes, confirmed, that is a new bug.

**Reply by dnewman-gpsw** [reply]
Found the bug, it will be fixed in future Labs builds.
